autoroom/autoroom.py

```diff
@@ -557,6 +558,13 @@ async def _process_autoroom_create(
                 reason="AutoRoom: New legacy text channel needed.",
                 overwrites=perms.overwrites if perms.overwrites else {},
```

Comment:
Is it possible to have `topic=text_channel_topic,` set here instead of a second API call to edit the channel? Depends on if `guild.create_text_channel()` allows us to send a None or empty string if there isn't a topic. Or, we could have the `guild.create_text_channel()` arguments set in a dict, and then add the `topic` key if set.

Reply:
Yeah i got it working with setting the default value to an empty string. Now everything works but only needs one api call.